### PR TITLE
Added pointer on putting whole solution in guide articles

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: docs/style-guide-for-guide-articles.md [hunk 1]
```diff
@@ -153,6 +153,10 @@ You can even embed relevant YouTube videos and interactive [REPL.it](https://rep
 
 Don't use emojis or emoticons in the Guide. freeCodeCamp has a global community, and the cultural meaning of an emoji or emoticon may be different around the world. Also, emojis can render differently on different systems.
 
+## Solutions
+
+In the solution which you propose for the guide, the full code should be given i.e. the seed code with the changes which have been made to achieve the desired result.
```
Comment (RandellDawson) [inline docs/style-guide-for-guide-articles.md]:
```suggestion
When proposing a solution for curriculum challenge related article, the full code should be given.  This includes all the original seed code plus any changes needed to pass the all the challenge's tests.
```

File: docs/style-guide-for-guide-articles.md [hunk 2]
```diff
@@ -153,6 +153,10 @@ You can even embed relevant YouTube videos and interactive [REPL.it](https://rep
 
 Don't use emojis or emoticons in the Guide. freeCodeCamp has a global community, and the cultural meaning of an emoji or emoticon may be different around the world. Also, emojis can render differently on different systems.
 
+## Solutions
```
Comment (RandellDawson) [inline docs/style-guide-for-guide-articles.md]:
```suggestion
## Curriculum Challenge Solutions
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@thecodingaviator What do you think about the following suggestions?

**Comment by thecodingaviator** [comment]
@RandellDawson Those were definitely better than what I had proposed, I made some minor corrections to it and it should look better now

File: docs/style-guide-for-guide-articles.md [hunk 3]
```diff
@@ -153,6 +153,10 @@ You can even embed relevant YouTube videos and interactive [REPL.it](https://rep
 
 Don't use emojis or emoticons in the Guide. freeCodeCamp has a global community, and the cultural meaning of an emoji or emoticon may be different around the world. Also, emojis can render differently on different systems.
 
+## Curriculum Challenge Solutions
+
+When proposing a solution for a curriculum challenge related article, the full code should be given. This includes all the original seed code plus any changes needed to pass the all the challenge's tests.
```
Comment (Manish-Giri) [inline docs/style-guide-for-guide-articles.md]:
```suggestion
When proposing a solution for a curriculum challenge related article, the full code should be given. This includes all the original seed code plus any changes needed to pass all the challenge tests.
```

**Review by Manish-Giri — CHANGES_REQUESTED** [review]
Suggested some grammatical changes.

File: docs/style-guide-for-guide-articles.md (hunk 3 above)
Comment (RandellDawson) [reply docs/style-guide-for-guide-articles.md]: I like @Manish-Giri's suggestion.

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
@thecodingaviator Still awaiting some requested change on on a couple ealier PRs before merging this one.

**Review by Nirajn2311 — APPROVED** [review]
LGTM 👍

**Comment by Nirajn2311** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

PS - @RandellDawson Since all other previous PRs were merged, I merged this one.

**Pull request merged**